components/infobox/wikis/leagueoflegends/infobox_league_custom.lua

```diff
@@ -0,0 +1,99 @@
+---
+-- @Liquipedia
+-- wiki=leagueofledends
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Template = require('Module:Template')
+
+local _args
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.liquipediaTierHighlighted = CustomLeague.liquipediaTierHighlighted
+	league.appendLiquipediatierDisplay = CustomLeague.appendLiquipediatierDisplay
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {args.team_number}
+	})
+	table.insert(widgets, Cell{
+		name = 'Players',
+		content = {args.participants_number}
+	})
+	return widgets
+end
+
+function CustomLeague:appendLiquipediatierDisplay()
+	if Logic.readBool(_args.riotpremier) then
+		return ' ' .. Template.safeExpand(mw.getCurrentFrame(), 'Riot/infobox')
```

Comment:
that template was created 3 days ago
so far it is only used in the standardized infobox
why not just write the function here?


```{{#vardefine:riot time|{{#time:U|{{{date|{{#replace:{{#replace:{{#explode: {{#var:date|{{#var:edate|{{#var:sdate|{{CURRENTYEAR}}-{{CURRENTMONTH}}-{{CURRENTDAY2}}}}}}}}|<}}|-XX|}}|-??|}}}}}}}}}<!-- this variable name needs to be unique --><!--
-->{{#ifexpr:{{#time:U|2019-04-16}} < {{#var:riot time}}|[[File:Riot Games 2020 logo.png|x12px|link=Riot Games|Premier Tournament held by Riot]]}}<!--
-->{{#ifexpr:{{#time:U|2019-04-16}} > {{#var:riot time}}|[[File:riotgameslogo std.png|x12px|link=Riot Games|Premier Tournament held by Riot]]}}<!--
--><noinclude>[[Category:Historical Small League Icon Templates]]</noinclude>```

Reply:
Added hardcoded icon has that's what the Current Live Template does.